lib/schema_web/templates/layout/app.html.eex

```diff
@@ -204,25 +195,27 @@ limitations under the License.
       <li id="base_event_id" class="nav-item">
         <a class="nav-link" href='<%= Routes.static_path(@conn, "/base_event") %>'>Base Event</a>
       </li>
-      <div class="navbar-text">|</div>
       <li id="dictionary_id" class="nav-item">
         <a class="nav-link" href='<%= Routes.static_path(@conn, "/dictionary") %>'>Dictionary</a>
       </li>
-      <div class="navbar-text">|</div>
       <li id="objects_id"  class="nav-item">
         <a class="nav-link" href='<%= Routes.static_path(@conn, "/objects") %>'>Objects</a>
       </li>
-      <li id="data_types_id" class="nav-item">
-        <a class="nav-link" href='<%= Routes.static_path(@conn, "/data_types") %>'>Data Types</a>
-      </li>
-      <div class="navbar-text">|</div>
-      <li id="guidelines_id" class="nav-item">
-        <a class="nav-link" href='<%= Routes.static_path(@conn, "/guidelines") %>'>Guidelines</a>
-      </li>
+
       <div class="navbar-text">|</div>
-      <li id="apidoc_id" class="nav-item">
-        <a class="nav-link" target="_blank" href='<%= Routes.static_path(@conn, "/doc") %>'>API Doc</a>
+
+      <li id=resources_id class="nav-link">
+        <div class="dropdown">
+          Resources
+          <div class="dropdown-content">
+            <a class="nav-link" href="https://github.com/ocsf/ocsf-docs/blob/main/Understanding%20OCSF.pdf">Understanding OCSF</a>
```

Comment:
Might just be a nitpick, but could we have `target="_blank"` to any resources that are not on `schema.ocsf.io`?

Reply:
good catch! adding it in